app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationViewModel.kt

```diff
@@ -595,5 +623,18 @@ sealed class CompleteRegistrationAction {
          * Indicates a generated password has been received.
          */
         data class GeneratedPasswordResult(val generatedPassword: String) : Internal()
+
+        /**
+         * Indicates registration was successful and will now attempt to login and unlock the vault.
+         */
+        data class AttemptLogin(val captchaToken: String?) : Internal()
```

Comment:
Do you think it's worth documenting here and/or in `ReceiveLoginResult` what happens when the `captchaToken` is null? It is not intuitive to me as someone who didn't work in the auth flow.